MACHINE-LEARNING-SERVICE/src/models/objectML.js

```diff
@@ -0,0 +1,29 @@
+const cocoSsd = require('@tensorflow-models/coco-ssd');
+const tf = require('@tensorflow/tfjs-node');
+const fs = require('fs').promises;
+class ObjectML{
```

Comment:
Add line break like this:
const fs = require('fs').promises;
(break like) 
class ObjectML{